keyboards/waldo/rev1/config.h

```diff
@@ -0,0 +1,8 @@
+#ifndef REV5_CONFIG_H
+#define REV5_CONFIG_H
+
+#include "config_common.h"
```

Comment:
This include is redundant because `keyboards/waldo/config.h` includes this file already.

```suggestion
```